src/app/(proper_react)/(redesign)/(public)/layout.tsx

```diff
@@ -7,29 +7,50 @@ import { PublicShell } from "./PublicShell";
 import {
   getAcceptLangHeaderInServerComponents,
   getL10n,
+  getL10nBundles,
 } from "../../../functions/l10n/serverComponents";
 import { headers } from "next/headers";
 import { getCountryCode } from "../../../functions/server/getCountryCode";
 import { getLocale } from "../../../functions/universal/getLocale";
 import { getExperiments } from "../../../functions/server/getExperiments";
 import { getExperimentationId } from "../../../functions/server/getExperimentationId";
 import { getEnabledFeatureFlags } from "../../../../db/tables/featureFlags";
+import { getServerSession } from "../../../functions/server/getServerSession";
+import * as Sentry from "@sentry/nextjs";
 
 export default async function Layout(props: { children: ReactNode }) {
-  const headersList = await headers();
-  const countryCode = getCountryCode(headersList);
   const enabledFeatureFlags = await getEnabledFeatureFlags({
     isSignedOut: true,
   });
   const l10n = getL10n(await getAcceptLangHeaderInServerComponents());
-  const experimentationId = await getExperimentationId(null);
+  const currentLocale = getLocale(
+    getL10nBundles(await getAcceptLangHeaderInServerComponents()),
+  );
```

Comment:
Can't we just pass in the `l10n` object we already have here?

```suggestion
  const currentLocale = getLocale(l10n);
```

Reply:
Yes! Updated in [c30e1f7](https://github.com/mozilla/blurts-server/pull/5549/commits/c30e1f7c4b3752b8f6a256f7fb21aeefa6b24b0c).